pbsmetrics/prometheus/prometheus.go

```diff
@@ -438,6 +486,15 @@ func requestTypesAsString() []string {
 	return output
 }
 
+func impTypesAsString() []string {
```

Comment:
I don't see this function being called anywhere. Can we please remove it if it isn't needed?

Reply:
That function is a left over of a previous version of this very PR. Removed.